src/category_theory/abelian/transfer.lean

```diff
@@ -140,7 +140,18 @@ local attribute [simp] cokernel_iso coimage_iso_image coimage_iso_image_aux
 -- Happily it's little effort: our `[ext]` and `[simp]` lemmas only need a little guidance.
```

Comment:
This seems to not really be true any more!

Reply:
should I just remove this comment.